docs/platform/manager/a-classic-to-cloud-manager-overview-guide/index.md

```diff
@@ -0,0 +1,295 @@
+
+### Find Volumes
+In Classic Manager, Volumes were found within a Linode's Dashboard tab. In Cloud Manager, Volumes are their own top-level menu item in the sidebar.
+
+![Cloud Manager Volumes](classic-to-cloud-cloud-manager-volumes.png "Cloud Manager Volumes")
+
+{{< note >}}
+In Cloud Manager, you can also access Volumes from a [Linode's detail page](/docs/platform/block-storage/how-to-use-block-storage-with-your-linode/#add-a-volume-from-the-linode-detail-page).
+{{</ note >}}
+
+## Object Storage
+
+The **Object Storage** section of Cloud Manager gives you access to Linode’s [Object Storage service](https://www.linode.com/products/object-storage/) which is a globally-available, S3-compatible method for storing and accessing data. The Object Storage service is not available in Classic Manager.
+
+To learn how to begin using Object Storage, view our [How to Use Linode Object Storage](/docs/platform/object-storage/how-to-use-object-storage/) guide. To access all available Object Storage guides, see the [Object Storage section](/docs/platform/object-storage/) of our documentation site.
+
+## NodeBalancers
+
+Linode's [NodeBalancers service](https://www.linode.com/products/nodebalancers/) provides load balancing for your applications and services ensuring that they are highly available for users. To learn how to get started with NodeBalancers using Cloud Manager, see our [Getting Started with NodeBalancers](/docs/platform/nodebalancer/getting-started-with-nodebalancers/#additional-nodebalancers-and-features) guide.
+
+## Domains (DNS Manager)
+The DNS Manager allows you to control and manage your domains. In Cloud Manager, the DNS Manager is located in the **Domains** link in the sidebar.
+
+![Cloud Manager Domains](classic-to-cloud-cloud-manager-domains.png "Cloud Manager Domains")
+
+For more information on Cloud Manager's DNS Manager, see the following guides:
+
+- [DNS Manager](/docs/platform/manager/dns-manager/).
+- [Common DNS Configurations](/docs/networking/dns/common-dns-configurations/)
+- [Configure Your Linode for Reverse DNS](/docs/networking/dns/configure-your-linode-for-reverse-dns/)
+
+### Zone Files
+
+Cloud Manager does not include the **Check Zone** and **Zone File** features, since it automatically ensures that your Domain's zone file does not contain any errors:
+
+- When creating a Zone File for a Domain, the [Linode API v4](https://developers.linode.com/api/v4) checks for any errors that may exist. If an error is found, the Cloud Manager will respond with the corresponding error. This means that the Cloud Manager will not allow you to create an invalid zone file.
+
+- Once your Domain and corresponding Zone File is created, you can use the `dig` command to further verify that each domain record contains the information you expect, for example:
+
+      dig example.com
```

Comment:
These for some reason in the Netlify preview don't show up as code blocks.

Reply:
I could reproduce locally...just pushed up a fix.